platform/src/main/java/platform/pageobjects/Header/Header.java

```diff
@@ -0,0 +1,362 @@
+package platform.pageobjects.Header;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import base.NewBaseClass;
+import utils.CommonFunctions;
+
+/**
+ * @Author : Amogh Nanal
+ * @Date : 22 Jul 2021
+ * @Description: Header Links and their Visibility
+ */
+
+public class Header extends NewBaseClass {
+
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(Header.class.getName());
+	int waitTime = 1;
+
+	@Override
+	public void closeZkaiPopup() {
+		if (CommonFunctions.waitForVisiblity(zkai_popup, waitTime)) {
+			zkai_popupCloseButton.click();
+			log.info("Close Zkai pop up");
+		}
+	}
+
+	public Header(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//div[@class='modal-content']")
+	public WebElement zkai_popup;
+
+	@FindBy(xpath = "//button[@aria-label='Close']")
+	public WebElement zkai_popupCloseButton;
+
+	// Xpaths for the elements
+
+	@FindBy(xpath = "//a[@class='logo logo-white']")
+	public WebElement stagiaLogo;
+
+	@FindBy(xpath = "//a[@class='menu__link js-not-link'][contains(text(),'受験の窓口')]")
+	public WebElement examinationWindowLink;
+
+	@FindBy(xpath = "//a[@class='menu__link js-not-link'][contains(text(),'学習の窓口')]")
+	public WebElement learningWindowLink;
+
+	@FindBy(xpath = "//a[@class='menu__link js-not-link'][contains(text(),'情報の窓口')]")
+	public WebElement informationWindowLink;
+
+	@FindBy(xpath = "//a[@href='/product-list']")
+	public WebElement productListLink;
+
+	@FindBy(xpath = "//a[@class='link link--smaller']")
+	public WebElement singUpLink;
+
+	@FindBy(xpath = "//a[@class='button button--default button--medium button--header']")
+	public WebElement loginLink;
+
+	// Xpaths for the Textboxes and Buttons
+
+	@FindBy(xpath = "//input[@id='email']")
+	public WebElement emailTextBox;
+
+	@FindBy(xpath = "//input[@id='entryIdentifier']")
+	public WebElement EIDTextbox;
+
+	@FindBy(xpath = "//button[@type='submit']")
+	public WebElement sendConfirmationEmailButton;
+
+	@FindBy(xpath = "//span[contains(text(),'ログイン')]")
+	public WebElement loginButton;
+
+	// Xpaths for the Sub Menus
+
+	@FindBy(xpath = "/html/body/div/div[1]/div[1]/div/div[1]/ul/li[1]/div/ul/li[1]/a/span")
```

Comment:
XPath not to use4d absolute xpath. It will relative xpath.

Reply:
Done Changes as requested